source/includes/_investors.md

```diff
@@ -616,16 +616,35 @@ Authorization: Basic ...
 HTTP/1.1 200 OK
 Content-Type: application/json
 
-{
-  "status" : "DOCUMENTS_REQUIRED"
-}
+    "documentsRequired" : [ "PROOF_OF_ADDRESS", "PROOF_OF_IDENTITY" ],
```

Comment:
Did we lose the opening and closing braces here?

Reply:
The payload in the docs was completley incorrect. I've updated it to be actually whats passed back

Reply:
So we don't pass back valid JSON?

Reply:
There's no `{ ... }` around the entire example

Reply:
Oh sorry misunderstood I thought you meant under status. My error